prime-router/settings/prod/0061-nj-suppress-obx-18.yml

```diff
@@ -0,0 +1,244 @@
+---
+- name: "nj-doh"
+  description: "New Jersey Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "NJ"
+  countyName: null
+  filters:
+    - topic: "covid-19"
+      jurisdictionalFilter:
+        - "orEquals(ordering_facility_state, NJ, patient_state, NJ)"
+        - "matches(processing_mode_code, P)"
+      qualityFilter:
+        - "hasAtLeastOneOf(order_test_date,specimen_collection_date_time,test_result_date)"
+        - "hasAtLeastOneOf(patient_street,patient_zip_code,patient_phone_number,patient_email)"
+        - "hasValidDataFor(message_id,equipment_model_name,specimen_type,test_result,patient_last_name,patient_first_name,patient_dob,ordering_provider_id,ordering_facility_name)"
+        - "isValidCLIA(testing_lab_clia)"
+      routingFilter: null
+      processingModeFilter: null
+  senders: []
+  receivers:
+    - name: "elr"
+      organizationName: "nj-doh"
+      topic: "covid-19"
+      customerStatus: "active"
+      translation: !<HL7>
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "ELRS"
+        receivingApplicationOID: "2.16.840.1.113883.3.1299.5.1.6.1"
+        receivingFacilityName: "NJDOH"
+        receivingFacilityOID: "2.16.840.1.113883.3.1299"
+        messageProfileId: null
+        replaceValue: {}
+        reportingFacilityName: "CDC PRIME"
+        reportingFacilityId: "11D2030855"
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: "OBX-18"
```

Comment:
Looks like only nj-doh.elr-secondary was updated to suppress all the subfields of OBX-18 in the yamlitos in /staging and /prod. Need to update the HL7 suppression for nj-doh.elr and possibly the other NJ receivers.